res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1036,8 +1087,8 @@ NumarkMixtrack3.initButtonsObjects = function() {
         NumarkMixtrack3.decks["D"+i].ShiftButtonControl =
             new SingleDoubleBtn(NumarkMixtrack3.OnShiftButton);
         NumarkMixtrack3.decks["D"+i].ShiftedPFLButtonControl =
-            new SingleDoubleBtn(NumarkMixtrack3.OnShiftedPFLButton);
-    }
+            new SingleDoubleBtn(NumarkMixtrack3.OnShiftedPFLButton);       
+    }    
```

Comment:
I don't know about Notepad++, but KWrite/Kate/KDevelop have an option to show dots to indicate trailing spaces. That helps with noticing them before git does.